.github/workflows/run_edps.yaml

```
@@ -54,18 +54,37 @@ jobs:
           export SOF_DATA="$(pwd)/METIS_Pipeline_Test_Data/small202402/outputSmall/"
           export SOF_DIR="$(pwd)/METIS_Pipeline_Test_Data/small202402/sofFiles/"
           export PYESOREX_OUTPUT_DIR="$SOF_DATA"
+          
+          # LIST RECIPES
           pyesorex --recipes
 
           # DET RECIPES
           pyesorex metis_det_lingain "${SOF_DIR}/metis_det_lingain.lm.sof"
           pyesorex metis_det_dark "${SOF_DIR}/metis_det_dark.lm.sof"
           
           # IMG LM RECIPES
+          pyesorex metis_lm_img_cal_distortion  "${SOF_DIR}/metis_lm_img_distortion.sof"
           pyesorex metis_lm_img_flat  "${SOF_DIR}/metis_lm_img_flat.lamp.sof"
           pyesorex metis_lm_img_basic_reduce "${SOF_DIR}/metis_lm_img_basic_reduce.std.sof"
+          pyesorex metis_lm_img_basic_reduce "${SOF_DIR}/metis_lm_img_basic_reduce.sci.sof"
+          pyesorex metis_lm_img_basic_reduce "${SOF_DIR}/metis_lm_img_basic_reduce.sky.sof"
+          pyesorex metis_lm_img_background "${SOF_DIR}/metis_lm_img_background.std.sof"
+          pyesorex metis_lm_img_background "${SOF_DIR}/metis_lm_img_background.sci.sof"
+          pyesorex metis_lm_img_std_process "${SOF_DIR}/metis_lm_img_std_process.sof"
+          # pyesorex metis_lm_img_calibrate "${SOF_DIR}/metis_lm_img_calibrate.sof"
+          # pyesorex metis_lm_img_sci_postproces "${SOF_DIR}/metis_lm_img_sci_postprocess.sof"
 
           # IFU RECIPES
+          pyesorex metis_ifu_distortion "${SOF_DIR}/metis_ifu_distortion.sof"
+          pyesorex metis_ifu_wavecal "${SOF_DIR}/metis_ifu_wavecal.sof"
           pyesorex metis_ifu_rsrf "${SOF_DIR}/metis_ifu_rsrf.sof"
+          pyesorex metis_ifu_reduce "${SOF_DIR}/metis_ifu_reduce.std.sof"
+          pyesorex metis_ifu_reduce "${SOF_DIR}/metis_ifu_reduce.sci.sof"
+          pyesorex metis_ifu_telluric "${SOF_DIR}/metis_ifu_telluric.std.sof"
+          pyesorex metis_ifu_telluric "${SOF_DIR}/metis_ifu_telluric.sci.sof"
+          pyesorex metis_ifu_telluric "${SOF_DIR}/metis_ifu_telluric.sci.sof"
```

You got the same one twice, maybe you meant
```suggestion
          pyesorex metis_ifu_telluric "${SOF_DIR}/metis_ifu_telluric.sky.sof"
```